index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfoli</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navbar">
+       <a class="logo">My Logo</a>
+       <img src="Union.png" alt="menubar">
```

Comment:
- [ ] Due to HTML and CSS best practices being a requirement, kindly wrap your `img` that represents the hamburger menu, inside a `<nav></nav>`, note: only the hamburger menu image should be wrapped, not the logo anchor